### DISCUSSION TITLE
input插件中ebpf功能启动报错

### DISCUSSION THREAD

**Discussion by ZZZZZZTong** [alibaba/ilogtail#1231] [Help]
按照[用户手册](https://ilogtail.gitbook.io/ilogtail-docs/data-pipeline/input/metric-observer)升级了内核以及下载了so文件，启动ilogtai后还是报错
错误日志：[error]	[28554]	/workspaces/ilogtail/core/observer/network/sources/ebpf/EBPFWrapper.cpp:386		init ebpf:failed    error:ebpf_init func failed
自己排错定位到： 
int err = g_ebpf_init_func(&btfPath.at(0),
                               static_cast<int32_t>(btfPath.size()),
                               &soPath.at(0),
                               static_cast<int32_t>(soPath.size()),
                               cleanup_offset,
                               update_conn_offset,
                               disable_process_offset,
                               update_role_offset);
 int32_t ebpf_init(char* btf,
                  int32_t btf_size,
                  char* so,
                  int32_t so_size,
                  long uprobe_offset,
                  long upca_offset,
                  long upps_offset,
                  long upcr_offset);
 看到提示说* @btf btf路径包括btf文件全名, 传NULL就默认在/usr/lib/vmlinux-**
 发现自己的linux机器上/usr/lib 路径下并没有vmlinux-**
 首先就考虑错误应该是这个路径未找到
请问这个btf文件是什么？该如何下载？

**Comment by EvanLjp** [comment]
uname -r 提供一下

**Reply by ZZZZZZTong** [reply]
5.4.260-1.el7.elrepo.x86_64

**Reply by ZZZZZZTong** [reply]
请问大佬，有解决这个问题的思路吗？我今天重新创了一个阿里云ECS实例，centOS7 也将内核版本升级到5.4.260-1.el7.elrepo.x86_64 ，还是一样的在ebpf_init func failed。
